src/main/java/org/openhab/tools/analysis/checkstyle/AboutHtmlCheck.java

```diff
@@ -26,30 +28,35 @@
 
 /**
  * Checks if an about.html file is valid.
+ * Verifies if the about.html file is added to the build.properties file
  *
- * @author Petar Valchev
- *
+ * @author Petar Valchev - Initial Implementation
+ * @author Svilen Valkanov - Add check for inclusion in build.properties file
  */
 public class AboutHtmlCheck extends AbstractStaticCheck {
```

Comment:
Wouldn't it make sense to add this to check which already processes `build.properties`-files now the method `processFiltered` has to process both html files and properties-files